### PR TITLE
Update variation test database 100 

### PR DESCRIPTION
**Pull request by dglemos**
Update variation test database. 

variation.t pass

removed deprecated ancestral_allele

### CODE REVIEW

**Comment by tgrego** [comment]
lib/EnsEMBL/REST/Model/Variation.pm also seems to have been modified, which is unusual for test db patching...

**Comment by dglemos** [comment]
I'm going to revert those two changes.

File: t/variation.t [hunk 1]
```diff
@@ -118,7 +118,8 @@ is_json_POST($base,$post_data,$expected_result,"Try to POST list of variations")
         'strand' => 1,
         'coord_system' => 'chromosome',
         'allele_string' => 'C/T',
-        'start' => 25744274
+        'start' => 25744274,
+        'ancestral_allele' => undef,
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be added?

File: t/variation.t [hunk 2]
```diff
@@ -164,7 +164,8 @@ my $publication_output =
         "strand" => 1,
         "coord_system" => "chromosome",
         "allele_string" => "C/A",
-        "start" => 103937974 
+        "start" => 103937974, 
+        "ancestral_allele" => undef,
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be added?

File: t/variation.t [hunk 3]
```diff
@@ -139,7 +140,6 @@ is_json_POST($base,$post_data,$expected_result,"Try to POST list of variations")
     'var_class' => 'somatic SNV',
     'synonyms' => [],
     'evidence' => [],
-    'ancestral_allele' => undef,
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be removed?

File: t/variation.t [hunk 4]
```diff
@@ -210,7 +210,6 @@ my $publication_output =
   is($variants_json->[0]->{MAF},$publication_output->[0]->{MAF} ,'PMID MAF key'); # Interestingly these are not numerically equal under certain testing conditions...
   is($variants_json->[0]->{ambiguity}, $publication_output->[0]->{ambiguity} ,'PMID ambiguity key');
   is($variants_json->[0]->{var_class}, $publication_output->[0]->{var_class} ,'PMID source key');
-  ok(!$variants_json->[0]->{ancestral_allele},'PMID ancestral_allele key not set');
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be removed?

File: t/variation.t [hunk 5]
```diff
@@ -249,7 +248,6 @@ my $publication_output =
   is($variants_json->[0]->{MAF},$publication_output->[0]->{MAF} ,'PMID MAF key');
   is($variants_json->[0]->{ambiguity}, $publication_output->[0]->{ambiguity} ,'PMID ambiguity key');
   is($variants_json->[0]->{var_class}, $publication_output->[0]->{var_class} ,'PMID source key');
-  ok(!$variants_json->[0]->{ancestral_allele},'PMID ancestral_allele key not set');
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be removed?

File: t/variation.t [hunk 6]
```diff
@@ -179,7 +180,6 @@ my $publication_output =
     "evidence" => 
       [
       ],
-    "ancestral_allele" => undef,
```
Comment (tgrego) [inline t/variation.t]: is this supposed to be removed?

File: t/variation.t (hunk 1 above)
Comment (dglemos) [reply t/variation.t]: I deleted this by mistake. Then I added it according to PR #404 (which I missed before I PR mine). I'm not adding the ancestral_allele, I'm just fixing it.

Comment (tgrego) [reply t/variation.t]: I stand corrected, it's just swapping places in the hash...
this solves the 2 insertions and deletions of `'ancestral_allele' => undef`

Comment (tgrego) [reply t/variation.t]: I stand corrected, it's just swapping places in the hash...
this solves the 2 insertions and deletions of `'ancestral_allele' => undef`

File: t/variation.t (hunk 4 above)
Comment (dglemos) [reply t/variation.t]: This is also removed in #404

File: t/variation.t (hunk 5 above)
Comment (dglemos) [reply t/variation.t]: This is also removed in #404

File: t/variation.t (hunk 4 above)
Comment (tgrego) [reply t/variation.t]: weird, don't know why github shows it as a change...

**Pull request merged**